projects/packages/search/src/instant-search/components/search-sort.jsx

```diff
@@ -35,6 +35,7 @@ export default class SearchSort extends Component {
 						{ __( 'Sort:', 'jetpack-search-pkg' ) }
 					</label>
 					<select
+						className="jetpack-instant-search__search-sort-select"
 						id="jetpack-instant-search__search-sort-select"
```

Comment:
It should be okay but would it be better to keep the id for backward compatibility?

Reply:
I have done I think? It has both a className and an id.

Reply:
oops my bad 👍

Reply:
🙂